### PR TITLE
informative error in Repository APIs

### PR DESCRIPTION
**Pull request by ravishetye**

### CODE REVIEW

**Comment by ravishetye** [comment]
@ad-47 if you. get time let us update the other error messages in similar fashion, in this PR

**Review by evgeniilezhnin — APPROVED** [review]

File: backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java [hunk 1]
```diff
@@ -545,7 +559,8 @@ public RepositoryEntity getRepositoryById(Session session, RepositoryIdentificat
               BranchEntity.class,
               new BranchEntity.BranchId(request.getBranch(), repository.getId()));
       if (branchEntity == null) {
-        throw new ModelDBException(ModelDBConstants.BRANCH_NOT_FOUND, Code.NOT_FOUND);
+        throw new ModelDBException(
+            ModelDBConstants.BRANCH_NOT_FOUND + request.getBranch(), Code.NOT_FOUND);
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java:563]:
```suggestion
            ModelDBConstants.BRANCH_NOT_FOUND + " " + request.getBranch(), Code.NOT_FOUND);
```

File: backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java [hunk 2]
```diff
@@ -613,13 +628,15 @@ public void deleteBranchByCommit(Long repoId, String commitHash) {
         query.setParameter("branch", request.getBranch());
         BranchEntity branchEntity = (BranchEntity) query.uniqueResult();
         if (branchEntity == null) {
-          throw new ModelDBException(ModelDBConstants.BRANCH_NOT_FOUND, Code.NOT_FOUND);
+          throw new ModelDBException(
+              ModelDBConstants.BRANCH_NOT_FOUND + request.getBranch(), Code.NOT_FOUND);
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java:632]:
```suggestion
              ModelDBConstants.BRANCH_NOT_FOUND + " " + request.getBranch(), Code.NOT_FOUND);
```

File: backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java [hunk 3]
```diff
@@ -613,13 +628,15 @@ public void deleteBranchByCommit(Long repoId, String commitHash) {
...
         }
         referenceCommit = branchEntity.getCommit_hash();
       } else {
         CommitEntity commit = session.get(CommitEntity.class, request.getCommitSha());
         if (commit == null) {
-          throw new ModelDBException(ModelDBConstants.COMMIT_NOT_FOUND, Code.NOT_FOUND);
+          throw new ModelDBException(
+              ModelDBConstants.COMMIT_NOT_FOUND + request.getCommitSha(), Code.NOT_FOUND);
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java:639]:
```suggestion
              ModelDBConstants.COMMIT_NOT_FOUND + " " + request.getCommitSha(), Code.NOT_FOUND);
```

File: backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java:563]: fixed https://github.com/VertaAI/modeldb/pull/455/files#diff-b54e5af072379e7b304a977ab286b085R204

File: backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java (hunk 2 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java:632]: Fixed https://github.com/VertaAI/modeldb/pull/455/files#diff-b54e5af072379e7b304a977ab286b085R204

File: backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java (hunk 3 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java:639]: Fixed https://github.com/VertaAI/modeldb/pull/455/files#diff-b54e5af072379e7b304a977ab286b085R205

**Review by anandjakhaniya — APPROVED** [review]
LGTM

**Pull request merged**